GridKit/AutomaticDifferentiation/DependencyTracking/Variable.hpp

```diff
@@ -254,6 +254,12 @@ namespace GridKit
       inline Variable& operator/=(const double& rhs);
       inline Variable& operator/=(const Variable& rhs);
 
+      // conversion operator
+      inline operator double() const
+      {
+        return value_;
+      }
```

Comment:
Why not call the function `value()`?

```suggestion
      inline operator value() const
      {
        return value_;
      }
```
It will not necessarily return a double floating precision type.

Reply:
This is a casting operator - it must be called the name of the type it returns. It is necessary for the `static_cast` used elsewhere.

Reply:
Thanks for clarifying, @alexander-novo.